packages/driver/src/cy/commands/request.ts

```diff
@@ -28,6 +28,7 @@ const REQUEST_DEFAULTS = {
   timeout: null,
   followRedirect: true,
   failOnStatusCode: true,
+  retryIntervals: [0, 100, 200, 200],
```

Comment:
Why are we adding retry intervals here and above?

Reply:
Previously this would inherit from the defaults in `server/lib/request`, but now it has to set its own. 

Automatic retries are still used for `cy.visit`'s `resolve:url` and `cy.request`, since those are not retriable by the browser. Fully removing automatic retries is a breaking change b/c we expose retryOnNetworkFailure and retryOnStatusCodeFailure as opts on cy.request, cy.visit